tests/loader_regression_tests.cpp

```diff
@@ -697,9 +699,7 @@ TEST(EnumeratePhysicalDevices, OneCall) {
 
 TEST(EnumeratePhysicalDevices, TwoCall) {
     FrameworkEnvironment env{};
-    auto& driver = env.add_icd(TestICDDetails(TEST_ICD_PATH_VERSION_2))
-                       .set_min_icd_interface_version(5)
-                       .add_instance_extension({VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME});
```

Comment:
I would prefer to not remove VK_KHR_get_physical_device_properties2 here (and later on in this file) because the loader looks for this extension when determining whether to do the 'linux stable order' logic, which should at least be executed by the tests, even if they aren't under test here.

Reply:
I see, I wasn't aware of the specific need for that here. I've simply removed it because it didn't seem to have an effect and in Vulkan SC that extension does not exist because its functionality is included in version 1.0, but we can employ other ways to get rid of those downstream.

Reply:
I've undone those changes, @charles-lunarg, please check if you have any other concerns. Thanks in advance!